DATA/EQUIPMENT/weapon_equip.ini

```diff
@@ -50970,3 +50970,135 @@ auto_turret = false
 turn_rate = 90
 LODranges = 0, 100, 200, 500, 1000
 lootable = true
+
+;; Kings Star and Stripes Set
+
+[Munition]
+nickname = kings_gun_laser_light01_ammo
+hp_type = hp_gun
+requires_ammo = false
+hit_pts = 2
+hull_damage = 96
+energy_damage = 0
+weapon_type = W_Laser
+one_shot_sound = fire_laser7
+munition_hit_effect = kings_laser_01_impact
+const_effect = kings_laser_01_proj
+lifetime = 0.8
+force_gun_ori = false
+mass = 1
+volume = 0
+lootable = false
+
+[Gun]
+nickname = kings_gun_laser_light01
+ids_name = 459450
+;res str
+; KING´S JUSTICE ($class)
+ids_info = 461042
+;res html
+; \m\bR7-X “Justice” Liberty Weapon\B

```

Comment:
No naming of King here? Or is the -X at the number the hint to be King's special gun?